### PR TITLE
Handle newlines

### PR DESCRIPTION
**Pull request by melcher**
Adds support for newlines at the end of the string. Just converted from paperclip and my tests all failed because it wasn't supported.

### CODE REVIEW

File: lib/carrierwave/base64/orm/activerecord.rb [hunk 1]
```diff
@@ -7,7 +7,8 @@ def mount_base64_uploader(attribute, uploader_class)
         mount_uploader attribute, uploader_class
 
         define_method "#{attribute}=" do |data|
-          if data.present? && data.end_with?("==")
+          if data.present?
```
Comment (melcher) [inline lib/carrierwave/base64/orm/activerecord.rb]: Also removed redundant ```end_with("==")``` check

**Comment by melcher** [comment]
Nokogiri install seems to be failing the build :(

**Comment by lebedev-yury** [comment]
thanks!

**Comment by lebedev-yury** [comment]
released a version 1.1

**Pull request merged**